src/modules/Crowbar.rb

```diff
@@ -49,22 +49,20 @@ def main
       @installed_file = "/opt/dell/crowbar_framework/.crowbar-installed-ok"
 
       # Names of default repositories available for current product
-      # For Cloud5, we support mixed SLE11/SLE12 environments
+      # For Cloud6, we support mixed SLE11/SLE12 environments
       @default_repos = [
         "SLE-Cloud",
         "SLE-Cloud-PTF",
-        "SUSE-Cloud-5-Pool",
-        "SUSE-Cloud-5-Updates",
+        "SUSE-OpenStack-Cloud-SLE11-6-Pool",
+        "SUSE-OpenStack-Cloud-SLE11-6-Updates",
         "SLES11-SP3-Pool",
         "SLES11-SP3-Updates",
         "SLE11-HAE-SP3-Pool",
         "SLE11-HAE-SP3-Updates",
         "SLES12-Pool",
         "SLES12-Updates",
-        "SLE12-Cloud-Compute",
-        "SLE12-Cloud-Compute-PTF",
```

Comment:
@jsuchome I'm not sure I got this right. One change that is being done for Cloud 6 is that SLE-Cloud and SLE-Cloud-PTF are both used for SLE11 and SLE12 (right now, since SLE11 will go away at some point). It should work fine because it will be in a suse-11.3/suse-12.0 section of the json, but I didn't find where the repos are matched to the section.